css/style.css

```diff
@@ -1,11 +1,159 @@
 
  /* Use the 'Roboto' font from Google. Already added to your HTML header */
 
+ body {
+  font-family: 'Roboto', sans-serif;
+  -webkit-font-smoothing: antialiased;
+}
+
+.nav-bar {
+  background: #fff;
+  position: fixed;
+  width: 100%;
+  border-bottom: 1px solid #DDD;
+  z-index: 9999;
+ 
+}
+
+.nav-bar .links {
+  height: 5.35rem;
+  width: 60%;
+  display: flex;
+  justify-content: center;
+  margin-top: 1.5rem;
+}
+
+.links {
+  padding: 0 1rem;
+  margin: 0 auto;
+  
+}
+
+.nav-bar .links li {
+  display: inline-flex;
+  color: #504d4b;
+  font-size: 1.5rem;
+  margin-left: 1.3rem;
+}
```

Comment:
Your nav is on the right track - you just need to align the `ul` to the right, and vertically centered (and the `li`s maybe a little smaller) and it should be fine!